### PR TITLE
Cenj 57 addition add icustomer repository interface to codebase

### PR DESCRIPTION
**Pull request by AbelRodriguezGuillen**
These files are CENJ-57 and CENJ-62

### CODE REVIEW

File: NJInsurancePlatform/Repositories/ICustomerRepository.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using NJInsurancePlatform.Models;
+
+namespace NJInsurancePlatform.Repositories
+{
+    public interface ICustomerRepository
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Repositories/ICustomerRepository.cs:5]: public interface ICustomerRepository: IDisposable

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/Repositories/ICustomerRepository.cs (hunk 1 above)
Comment (Marcus-Johnson) [reply NJInsurancePlatform/Repositories/ICustomerRepository.cs:5]: has using statement as well

Comment (AbelRodriguezGuillen) [reply NJInsurancePlatform/Repositories/ICustomerRepository.cs:5]: Thank you, I'll make the changes. I dont thing the merge I just approved was inheriting idisposable either

Comment (Marcus-Johnson) [reply NJInsurancePlatform/Repositories/ICustomerRepository.cs:5]: throw in the fix with this patch just check all repositories then

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/Repositories/ICustomerRepository.cs (hunk 1 above)
Comment (AbelRodriguezGuillen) [reply NJInsurancePlatform/Repositories/ICustomerRepository.cs:5]: will do

**Review by Marcus-Johnson — APPROVED** [review]

**Pull request merged**